server/app/routes/reviews.js

```diff
@@ -0,0 +1,42 @@
+var router = require('express').Router();
+
+var models = require('../db/models');
+
+var Review= models.Review;
+
+module.exports = router;
+
+router.param('reviewId', function(req, res,next,id){
+	Review.findById(id)
+	.then(review => req.review= review);
+});
+
+router.get('/', function (req, res, next) {
+  Review.findAll()
+  .then(reviews => res.json(reviews))
+  .catch(next);
+});
+
+
+router.get('/:reviewId', function (req, res, next) {
+  res.send(req.review);
```

Comment:
I think res.send will send it as json anyways. You guys know the benefit of one over the other?